pgcontents/pgmanager.py

```diff
@@ -79,15 +73,13 @@ def _checkpoints_class_default(self):
         return PostgresCheckpoints
 
     def _checkpoints_kwargs_default(self):
-        kw = super(PostgresContentsManager, self)._checkpoints_kwargs_default()
-        kw.update(
-            {
-                'db_url': self.db_url,
-                'user_id': self.user_id,
-                'create_user_on_startup': self.create_user_on_startup,
-            }
-        )
-        return kw
+        return {
+            'db_url': self.db_url,
+            'user_id': self.user_id,
+            'create_user_on_startup': self.create_user_on_startup,
+            'crypto': self.crypto,
+            'max_file_size_bytes': self.max_file_size_bytes,
+        }
```

Comment:
Ah, good catch.  I was thinking that the super call didn't do anything because `PostgresManagerMixin` didn't implement this, but it's provided by the `ContentsManager` base class to provide

```
    def _checkpoints_kwargs_default(self):
        return dict(
            parent=self,
            log=self.log,
        )
```